### PR TITLE
feat(customer): add remove from group, update group

### PR DESCRIPTION
**Pull request by srindom**
Depends on #6148 
**What**
- adds removeCustomerFromGroup
- adds updateCustomerGroup

### CODE REVIEW

File: packages/types/src/customer/service.ts [hunk 1]
```diff
@@ -66,6 +82,15 @@ export interface ICustomerModuleService extends IModuleService {
     sharedContext?: Context
   ): Promise<{ id: string }[]>
 
+  removeCustomerFromGroup(
+    groupCustomerPair: { customer_id: string; customer_group_id: string },
```
Comment (olivermrbl) [inline packages/types/src/customer/service.ts:87]: suggestion: If type added in prev. PR, use here

File: packages/customer/src/services/customer-module.ts [hunk 2]
```diff
@@ -313,6 +380,32 @@ export default class CustomerModuleService implements ICustomerModuleService {
...
+  ): Promise<void>
+  async removeCustomerFromGroup(
+    groupCustomerPairs: { customer_id: string; customer_group_id: string }[],
+    sharedContext?: Context
+  ): Promise<void>
+
+  @InjectTransactionManager("baseRepository_")
+  async removeCustomerFromGroup(
+    data:
+      | { customer_id: string; customer_group_id: string }
```
Comment (olivermrbl) [inline packages/customer/src/services/customer-module.ts]: Same

**Review by olivermrbl — APPROVED** [review]
LGTM!

File: packages/customer/src/services/customer-module.ts [hunk 3]
```diff
@@ -284,6 +284,73 @@ export default class CustomerModuleService implements ICustomerModuleService {
...
+    data: Partial<CustomerTypes.CreateCustomerGroupDTO>,
+    sharedContext?: Context
+  ): Promise<CustomerTypes.CustomerGroupDTO[]>
+  async updateCustomerGroup(
+    selector: CustomerTypes.FilterableCustomerGroupProps,
+    data: Partial<CustomerTypes.CreateCustomerGroupDTO>,
+    sharedContext?: Context
+  ): Promise<CustomerTypes.CustomerGroupDTO[]>
+
+  @InjectManager("baseRepository_")
```
Comment (adrien2p) [inline packages/customer/src/services/customer-module.ts]: todo: InjectTransactionManager

File: packages/customer/src/services/customer-module.ts [hunk 4]
```diff
@@ -284,6 +284,73 @@ export default class CustomerModuleService implements ICustomerModuleService {
...
+  async updateCustomerGroup(
+    groupIdOrSelector:
+      | string
+      | string[]
+      | CustomerTypes.FilterableCustomerGroupProps,
+    data: Partial<CustomerTypes.CreateCustomerGroupDTO>,
+    @MedusaContext() sharedContext: Context = {}
+  ) {
+    let updateData: CustomerTypes.UpdateCustomerGroupDTO[] = []
+    if (typeof groupIdOrSelector === "string") {
```
Comment (adrien2p) [inline packages/customer/src/services/customer-module.ts]: todo: isString

File: packages/customer/src/services/customer-module.ts [hunk 5]
```diff
@@ -284,6 +284,73 @@ export default class CustomerModuleService implements ICustomerModuleService {
...
+        ...data,
+      }))
+    }
+
+    const groups = await this.customerGroupService_.update(
+      updateData,
+      sharedContext
+    )
+
+    if (typeof groupIdOrSelector === "string") {
```
Comment (adrien2p) [inline packages/customer/src/services/customer-module.ts]: todo: isString

**Review by adrien2p — APPROVED** [review]
LGTM 🔥 couple of todo's

**Pull request merged**